src/components/Pricing/Pricing.tsx

```diff
@@ -602,7 +604,11 @@ const Pricing = ({
                                                                 )}
                                                                 {planFeature.limit && (
                                                                     <span className="opacity-75">
-                                                                        {planFeature.limit} {planFeature.unit}
+                                                                        {planFeature.key === 'organizations_projects' &&
+                                                                        posthog?.getFeatureFlag?.('two-project-limit')
+                                                                            ? 2
+                                                                            : planFeature.limit}{' '}
```

Comment:
I'd use renderInClient here as well so that people don't see the wrong thing initially